project/Rat.scala

```diff
@@ -62,6 +62,11 @@ object Rat {
     file("src/svelte/src/resources/fonts/RedHatMono-Regular.woff2"),
     // scalafmt-core files
     file(".git-blame-ignore-revs")
```

Comment:
Add a common after this line, that's why the CI is failing

```suggestion
    file(".git-blame-ignore-revs"),
```

Reply:
Fixed.